src/cdk-experimental/scrolling/virtual-scroll-viewport.scss

```diff
@@ -7,18 +7,18 @@ cdk-virtual-scroll-viewport {
 
 // Wrapper element for the rendered content. This element will be transformed to push the rendered
 // content to its correct offset in the data set as a whole.
-.cdk-virtual-scroll-content-wrapper {
+.cdk-virtual-scroll-content {
   position: absolute;
   top: 0;
   left: 0;
   will-change: contents, transform;
 }
 
-.cdk-virtual-scroll-orientation-horizontal .cdk-virtual-scroll-content-wrapper {
+.cdk-virtual-scroll-orientation-horizontal .cdk-virtual-scroll-content {
```

Comment:
Could this selector and the one below be nested under the `.cdk-virtual-scroll-orientation-`?